components/frontend/src/dashboard/CardDashboard.js

```diff
@@ -70,6 +70,7 @@ export function CardDashboard({ cards, initialLayout, saveLayout }) {
                 onDragStop={onDragStop}
                 preventCollision={true}
                 rowHeight={24}
+                style={{ zIndex: "0" }}  // Prevent cards from being shown above the settings panel after being clicked
```

Comment:
The extra bugfix? Strange that this happens

Reply:
I guess we didn't discover it before because all cards, except the Legend card, have a click handler.